### PR TITLE
docs: add browser support section to the docs

### LINKED ISSUE #293: Add a "Browser support" section in the documentation
The idea here is to add a section showing which browser we are supporting as extensions (shown as icons)

### PR DESCRIPTION
**Pull request by tomiiide**
Fixes #293 

## Screenshot

Screenshot 2022-12-13 at 5 04 03 PM

### CODE REVIEW

**Comment by luizstacio** [comment]
@tomiiide did you tested on edge?

**Comment by tomiiide** [comment]
> @tomiiide did you tested on edge?

Yes, I did.
Screenshot 2022-12-13 at 5 05 23 PM

**Comment by LuizAsFight** [comment]
any specific reason to use table/tr/td? it's kind of old stuff and hard to manipulate.. 

I would suggest changing it to flex containers, but not a requirement for approving this PR

File: packages/docs/docs/browser-support.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+|                                                                             | ![Chrome Logo](/icons/browser/chrome.png "Chrome Logo") Chrome | ![Brave Logo](/icons/browser/brave.png "Brave Logo") Brave | ![Microsoft Edge Logo](/icons/browser/edge.png "Microsoft Edge Logo") Microsoft Edge | ![Firefox Logo](/icons/browser/firefox.png "Firefox Logo") Firefox |
+| --------------------------------------------------------------------------- | -------------------------------------------------------------- | ---------------------------------------------------------- | ------------------------------------------------------------------------------------ | ------------------------------------------------------------------ |
+| ![Fuel Wallet Logo](/icons/fuel-logo-32.png "Fuel Logo") Fuel Wallet v0.0.2 | v108                                                           | v1.46 (Chromium v108)                                      | v108                                                                                 | Not supported yet                                                  |
```
[the remark this answers is not in the document]
Comment (tomiiide) [reply packages/docs/docs/browser-support.mdx:11]: yeah, we would need to update this manually, so I'll remove it.

**Comment by tomiiide** [comment]
> any specific reason to use table/tr/td? it's kind of old stuff and hard to manipulate..
> 
> I would suggest changing it to flex containers, but not a requirement for approving this PR

Yeah, I used a table element so that it corresponds with the markdown table format. I agree that a flex container makes more sense tho, so I'll update it.

**Review by LuizAsFight — APPROVED** [review]
LGTM

**Pull request merged**